Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scenes/SpatialUnderstandingExample.unity

```diff
@@ -1735,6 +1744,11 @@ Prefab:
       propertyPath: drawVisualMeshes
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 114000012149591842, guid: 2ed75ffdf9031c94e8bce4b3d17b9928,
+        type: 2}
+      propertyPath: RoomModel
+      value: 
+      objectReference: {fileID: 100006, guid: c35ec8112ca03e04e904b72932bc97c5, type: 3}
```

Comment:
This room model is Assets\HoloToolkit\SpatialMapping\Tests\Meshes\SRMesh.obj .  It's not a very interesting mesh, especially for spatial understanding.  Anyone opposed if I replace SRMesh.obj with one more interesting?